internal/pkg/aws/ecs/ecs.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package ecs contains utility functions for dealing with ECS repos
+package ecs
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+)
+
+type ecsClient interface {
+	DescribeTaskDefinition(input *ecs.DescribeTaskDefinitionInput) (*ecs.DescribeTaskDefinitionOutput, error)
+}
+
+// Service wraps an AWS ECS client.
+type Service struct {
+	ecs ecsClient
+}
+
+// New returns a Service configured against the input session.
+func New(s *session.Session) *Service {
+	return &Service{
+		ecs: ecs.New(s),
+	}
+}
+
+// EnvVars calls ECS API and returns the environment variables of the task definition.
+func (s Service) EnvVars(taskDefName string) (map[string]string, error) {
```

Comment:
Maybe we can just return a type like TaskDefinition, with some convenience methods on it. 

```golang
func (s Service) TaskDefinition(taskDefName string) (TaskDefinition, error) { ...}

type TaskDefinition aws.TaskDefinition

func (t TaskDefinition) EnvironmentVariables() map[string]string { ... }

```

The reason I say that is that I suspect we'll want other data off the TaskDefinition in the future, such as secrets or Hostname, or HealthCheck - and having a convenient datastructure, rather than a bunch of methods will simplify access (rather than having to worry about caching the result to describetask set).